lib/bot.js

```diff
@@ -481,7 +481,7 @@ Bot.prototype.newRoom = function (name, emails, isTeam) {
 
   // add room
   var newRoom = { title: name };
-  if (teamId) { newRoom.team = teamId; }
+  if (teamId) { newRoom.teamId = teamId; }
```

Comment:
Missing this definitely seems like it was a bug.  See my comment below about whether or not this team object in the bot object really adds value.  It seems like this could also have been implemented as:

```js
 // Validate team
  if (isTeam) {
    if (!this.room.teamId) {
      return when.reject(new Error('This room is not part of a Webex Teams Team'));
    }
  }

  // add room
  var newRoom = { title: name };
  if (isTeam) { newRoom.teamId = this.room.teamId; }
  return this.framework.webex.rooms.create(newRoom) 
```

The slight benefit of this approach is that we are minimizing the amount of "non webex DTOs" that we are adding to the bot object.